src/material-experimental/mdc-theming/_all-theme.scss

```diff
@@ -12,6 +12,7 @@
 @import '../mdc-progress-bar/progress-bar-theme';
 @import '../mdc-input/input-theme';
 @import '../mdc-form-field/form-field-theme';
+@import '../../material/core/core';
```

Comment:
I think the CI failures are reasonable. You are importing the `_core.scss` file while the target that provides it as `sass_library` is not part of the target dependencies. i.e. `theming_scss_lib`

Reply:
Will adding theming_scss_lib to the mdc indigo_pink_prebuilt pull in a bunch of unneeded standard material stuff, too?

Reply:
Not that I know of, but you should add the dependency to the `all-theme` target as that is the one declaring the `_all-theme.scss` file (where `material/core/core.scss` is imported).